OpenSim/Common/Reporter.h

```diff
@@ -113,6 +113,19 @@ class Reporter : public AbstractReporter {
     OpenSim_DECLARE_LIST_INPUT(inputs, InputT, SimTK::Stage::Report,
         "Variable list of quantities to be reported.");
 
+    //=============================================================================
+    // PUBLIC METHODS
+    //=============================================================================
+
+    // Allow overloading of updInput
+    using AbstractReporter::updInput;
```

Comment:
How come this was necessary? Could this go inside `updInput()`? Would this be necessary if you used `this->updInput("inputs")` instead?